gammapy/scripts/cta_sensitivity.py

```diff
@@ -0,0 +1,250 @@
+#-*- coding: utf-8 -*-
+
+"""
+Compute the CTA sensitivity for a fixed observation config
+Authors: J. Lefaucheur (LUTh), B. Khelifi (APC)
+Date: 31/05/2017
+Version: 0.1
+
+Note: For the moment, only the differential point-like sensitivity is computed
+ToDo: - make the computation for any source size
+      - compute the integral sensitivity
+      - Add options to use different spectral shape?
+
+Inputs:
+    - filename: str
+        IRF filename (full path, in general there are here: ../datasets/cta/)
+        e.g. ../datasets/cta/CTA-Performance-South-20170323/CTA-Performance-South-20deg-30m_20170323.fits
+    - livetime: u.h
+        Livetime (object with the units of time), e.g. 5*u.h
+Optionnal inputs:
+    - slope: float (>0)
+        Index of the spectral shape (Power-law)
+    - alpha: float
+        On/OFF normalisation
+    - sigma: float
+        minimun significance
+    - gamma_min: float
+        Minimum number of gamma-rays
+    - bkg_sys: float
+        Fraction of Background systematics relative to the number of ON counts
+    - verbosity: int
+        level of print out [int: 0/1/2(debug)]. In the debug mode, stats for each bin are printed and the computed \
+        sensitivity curve is superposed with the one stored in the IRF fits file
+
+"""
+
+# Inputs :
+
+# IRFs :
+#  - taux de fond en énergie reconstruite
+#  - matrice de dispersion en énergie
+#  - surface efficace en énergie vraie
+# Paramètres :
+#  - nsigma, alpha, temps, ngamma_min et systématiques fond
+
+# Taux de fond, calcul de nombre de gamma
+
+# calcul du flux avec la double intégrale et le nombre de gamma
+
+import astropy.units as u
+import numpy as np
+import matplotlib.pyplot as plt
+import sys
+
+from gammapy.stats import significance_on_off
+from gammapy.spectrum.models import PowerLaw
+from gammapy.spectrum.utils import CountsPredictor
+from gammapy.scripts import CTAPerf
+from gammapy.utils.scripts import get_parser, set_up_logging_from_args
+
+
+class cta_sensi_estim(object):
+    """
+    Parameters:
+        - filename: str
+            IRF filename (full path, in general there are here: ../datasets/cta/)
+            e.g. ../datasets/cta/CTA-Performance-South-20170323/CTA-Performance-South-20deg-30m_20170323.fits
```

Comment:
Don't put this example here. It's already not working any more, because you moved the file.
If you want to give an example in the class or a method docstring, use an `Examples` section.
Example how to do that: http://docs.gammapy.org/en/latest/api/gammapy.spectrum.CrabSpectrum.html

Or don't give an example here, but instead link to your notebook with the example like this:
http://docs.gammapy.org/en/latest/development/howto.html#link-to-a-notebook-in-gammapy-extra-from-the-docs